src/main/resources/db/changelog/changes/v1.1.0/insert-organizations-contact-info-entity-type.xml

```diff
@@ -147,13 +147,30 @@
               "dataType": {
                 "dataType": "arrayType",
                 "itemDataType": {
-                  "dataType": "stringType"
-                }
-              },
-              "queryable": true,
-              "valueGetter": "(SELECT array_agg(alias) FROM jsonb_array_elements_text(org.jsonb -> 'aliases') AS alias)",
-              "filterValueGetter": "lower(${tenant_id}_mod_organizations_storage.f_unaccent(org.jsonb->>'aliases'::text))",
-              "valueFunction": "lower(${tenant_id}_mod_organizations_storage.f_unaccent(:value))",
+                  "dataType": "objectType",
+                  "properties": [
+                      {
+                        "name": "value",
+                        "property": "value",
+                        "dataType": {"dataType": "stringType"},
+                        "queryable": true,
+                        "valueGetter": "( SELECT array_agg(elems.value ->> 'value') FROM jsonb_array_elements(org.jsonb -> 'aliases') AS elems)",
```

Comment:
don't need value getters here anymore, since the main valueGetter for `alias` itself works